administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -23,13 +24,27 @@ const BirthdayForm = ({
   const [touched, setTouched] = useState(false)
   const { birthday } = value
   const showErrorMessage = touched || showRequired
+  const projectConfig = useContext(ProjectConfigContext)
+
+  const isBirthdayHintEnabled = (): boolean => {
+    const today = PlainDate.fromLocalDate(new Date())
+    const underAge = today.subtract({ years: 16 })
+    return !!(birthday?.isAfter(underAge) && projectConfig.showBirthdayExtensionHint && !birthday.isAfter(today))
+  }
+
   const getErrorMessage = (): string | null => {
+    const today = PlainDate.fromLocalDate(new Date())
+
     if (!birthday) {
       return t('birthdayMissingError')
     }
-    if (birthday.isAfter(PlainDate.fromLocalDate(new Date()))) {
+    if (birthday.isAfter(today)) {
       return t('birthdayFutureError')
     }
+    if (isBirthdayHintEnabled()) {
```

Comment:
:wrench:
i think this hint should be handled separately, since its not an error